Rakefile

```diff
@@ -200,6 +200,17 @@ if defined?(Gem)
       end
     end
 
+    desc "Prepares for the patch release"
+    task :travis do
+      load "ci/release.rb"
+      sh <<-SH
+        echo "---\n:rubygems_api_key: #{ENV['RUBYGEMS_API_KEY']}" > ~/.gem/credentials
+        cat ~/.gem/credentials
```

Comment:
Does this need output here?

Reply:
I totally forgot about this, but seems a debug code to me

Reply:
I'll try to take a look in a few days.

Reply:
I just went over this again, and noted our conversation here. Good to see your avatar again. Hope things are good! 👋